cycode/cli/utils/path_utils.py

```diff
@@ -97,6 +97,10 @@ def change_filename_extension(filename: str, extension: str) -> str:
 def concat_unique_id(filename: str, unique_id: str) -> str:
     if filename.startswith(os.sep):
         # remove leading slash to join the path correctly
-        filename = filename[len(os.sep) :]
+        filename = filename[len(os.sep):]
 
     return os.path.join(unique_id, filename)
+
+
+def get_path_from_context(context: dict) -> str:
+    return context.params['path'] if 'path' in context.params else context.params['paths'][0]
```

Comment:
fixed context type; add make optional return type in case of missing path/paths

we need to add "import click" at the top

```suggestion
def get_path_from_context(context: click.Context) -> Optional[str]:
    path = context.params.get('path')
    if path is None and 'paths' in context.params:
        path = context.params['paths'][0]

    return path
```

Reply:
Actually that was my first implementation at the beginning and that is why I wouldn't used "context.params['path'] if 'path' in context.params else context.params['paths'][0]"

I'll return it back
Check it out

Reply:
I was so confused by your implementation because `project_path` is never will be `None` because of the default value set to array (`[None]`). and calling `[0]` from `path` and not from `paths`. But in the end, thank you so much for your patience and contribution!